### PR TITLE
Implement users signup

### PR DESCRIPTION
**Pull request by Somiel**
- [x] DB sheme development
- [x] Basic authorization implement (devise install/config)
- [x] Views refactoring (migrate to slim)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,6 @@
 /log/*
 !/log/.keep
 /tmp
+
+# Ignore rubymine project files
+/.idea/*
```
Comment (murderface13) [inline .gitignore]: add new line in the end of file, pls

Comment (Somiel) [inline .gitignore]: done

File: app/views/layouts/application.html.slim [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    = stylesheet_link_tag    'home', media: 'all', 'data-turbolinks-track' => true
+    = javascript_include_tag 'application', 'data-turbolinks-track' => true
+    = csrf_meta_tags
+  body
+    main class="main"
+      .navbar-fixed
+        nav
+          .nav-wrapper
+            = link_to "Sloboda Taxi", root_path, { class: 'brand-logo' }
+            ul class="right hide-on-med-and-down"
```
Comment (murderface13) [inline app/views/layouts/application.html.slim]: Move user profile links to partial, please

File: app/views/layouts/application.html.slim [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    footer class="page-footer footer-fixed"
+      .container
+        .row
+          .col.l6.s12
+            h5 class="white-text" Footer Content
+            p class="grey-text text-lighten-4" You can use rows and columns here to organize your footer content
+          .col.l4.offset-l2.s12
+            h5 class="white-text" Links
+      .footer-copyright
+        .container © 2014 Copyright Text
```
Comment (murderface13) [inline app/views/layouts/application.html.slim]: don't forget empty newline in the end :wink:

File: db/schema.rb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    t.integer  "sign_in_count",          default: 0,  null: false
+    t.datetime "current_sign_in_at"
+    t.datetime "last_sign_in_at"
+    t.string   "current_sign_in_ip"
+    t.string   "last_sign_in_ip"
+    t.datetime "created_at",                          null: false
+    t.datetime "updated_at",                          null: false
+    t.string   "type"
+    t.string   "first_name"
+    t.string   "last_name"
```
Comment (murderface13) [inline db/schema.rb]: I guess, user must have phone number - manager or driver can phone him if it needs

File: test/controllers/home_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (murderface13) [inline test/controllers/home_controller_test.rb]: Scaffold is here :smile:

File: public/assets/material-design-icons/LICENSE-04051e0a14494a4997cedd4e371ccc9e9262fcd278c9a96129c36b4277c4129d.txt [hunk 6]
```diff
@@ -0,0 +1,428 @@
+https://github.com/google/material-design-icons/blob/master/LICENSE
```
Comment (murderface13) [inline public/assets/material-design-icons/LICENSE-04051e0a14494a4997cedd4e371ccc9e9262fcd278c9a96129c36b4277c4129d.txt]: what is that file doing? why we need it?

File: test/controllers/home_controller_test.rb (hunk 5 above)
Comment (Somiel) [inline test/controllers/home_controller_test.rb]: That was automated controller generator (rails g controller..)

Comment (murderface13) [inline test/controllers/home_controller_test.rb]: That's some kind of scaffold :smirk: 

please, remove all files which you don't need

File: public/assets/material-design-icons/LICENSE-04051e0a14494a4997cedd4e371ccc9e9262fcd278c9a96129c36b4277c4129d.txt (hunk 6 above)
Comment (Somiel) [inline public/assets/material-design-icons/LICENSE-04051e0a14494a4997cedd4e371ccc9e9262fcd278c9a96129c36b4277c4129d.txt]: This is some license file from materialize gem, i think i can exclude it.

File: db/schema.rb (hunk 4 above)
Comment (Somiel) [inline db/schema.rb]: Done

File: test/controllers/home_controller_test.rb (hunk 5 above)
Comment (Somiel) [inline test/controllers/home_controller_test.rb]: Kinda done :)

**Comment by murderface13** [comment]
:+1:

**Pull request merged**